modules/storages/app/components/storages/admin/storage_view_component.html.erb

```diff
@@ -0,0 +1,165 @@
+<%=
+  render(Primer::Beta::BorderBox.new) do |component|
+    component.with_header(color: :default) do
+      grid_layout('op-storage-view--header', tag: :div, align_items: :center) do |grid|
```

Comment:
Why do you need a grid if there is only one element inside anyways?

Reply:
yeah, you're right-we don't need it 🧹